### PR TITLE
Remove Rule. prefix from rule names

### PR DESCRIPTION
**Pull request by brian-brazil**

### CODE REVIEW

File: src/com/carolinarollergirls/scoreboard/view/ScoreBoard.java [hunk 1]
```diff
@@ -69,30 +69,30 @@
...
-    public static final String RULE_STOP_PC_ON_OR = "Rule." + Clock.ID_TIMEOUT + ".StopPeriodClockOnOR";
-    public static final String RULE_STOP_PC_AFTER_TO_DURATION = "Rule." + Clock.ID_TIMEOUT + ".StopPeriodClockAfterTODuration";
-    public static final String RULE_INTERMISSION_DURATIONS = "Rule." + Clock.ID_INTERMISSION + ".Durations";
-    public static final String RULE_INTERMISSION_DIRECTION = "Rule." + Clock.ID_INTERMISSION + ".Direction";
-    public static final String RULE_AUTO_START = "Rule.ClockControl.AutoStart";
-    public static final String RULE_AUTO_START_JAM = "Rule.ClockControl.AutoStartType";
-    public static final String RULE_AUTO_START_BUFFER = "Rule.ClockControl.AutoStartBuffer";
-    public static final String RULE_AUTO_END_JAM = "Rule.ClockControl.AutoEndJam";
-    public static final String RULE_AUTO_END_TTO = "Rule.ClockControl.AutoEndTTO";
+    public static final String RULE_NUMBER_PERIODS = "" + Clock.ID_PERIOD + ".Number";
```
Comment (frank-weinberg) [inline src/com/carolinarollergirls/scoreboard/view/ScoreBoard.java]: The `"" + `-Prefixes should be removed

File: src/com/carolinarollergirls/scoreboard/defaults/DefaultClockModel.java [hunk 2]
```diff
@@ -321,6 +321,9 @@ public void setCountDirectionDown(boolean down) {
             Boolean last = new Boolean(countDown);
             countDown = down;
             scoreBoardChange(new ScoreBoardEvent(this, EVENT_DIRECTION, new Boolean(countDown), last));
+            if (last != down) {
+                setTime(getInvertedTime());
+            }
```
Comment (frank-weinberg) [inline src/com/carolinarollergirls/scoreboard/defaults/DefaultClockModel.java:329]: In the same spirit of keeping `timeElapsed` constant, we should change the time of a countdown clock, when the maximum time is changed. (Most common use case by far for changing maximum time is changing halftime duration after it started.)

File: src/com/carolinarollergirls/scoreboard/view/ScoreBoard.java (hunk 1 above)
Comment (brian-brazil) [reply src/com/carolinarollergirls/scoreboard/view/ScoreBoard.java]: Fixed

File: src/com/carolinarollergirls/scoreboard/defaults/DefaultClockModel.java (hunk 2 above)
Comment (brian-brazil) [reply src/com/carolinarollergirls/scoreboard/defaults/DefaultClockModel.java:329]: Done

**Pull request merged**